src/views/GovernanceReviewTeam/Notes/index.tsx

```diff
@@ -534,15 +547,22 @@ const Notes = () => {
                 }}
               >
                 <FieldGroup>
-                  <Label htmlFor="GovernanceReviewTeam-Note">
+                  <Label
+                    id="GovernanceReviewTeam-Note-label"
+                    htmlFor="GovernanceReviewTeam-Note"
+                  >
                     {t('notes.addNote')}
                   </Label>
-                  <Field
-                    as={TextAreaField}
+                  <RichTextEditorFormikField
                     id="GovernanceReviewTeam-Note"
-                    maxLength={2000}
                     name="note"
-                    className="easi-grt__note-field"
+                    height="5rem"
```

Comment:
This height seems small/not the height in the linked Figmafrom ticket - [https://www.figma.com/file/iGatgTnlzTuqW9CQFYHDam/TRB-Intake-e2e?type=design&node-id=4855-1528&mode=design&t=1MZypVhfPrKnln2l-0](url)

I would maybe doublecheck with UX

Reply:
Thanks for catching that. I was using existing heights.

Reply:
Using 405px from figma